commons/ava_testsuite/ava_test_suite.go

```diff
@@ -4,11 +4,17 @@ import (
 	"github.com/kurtosis-tech/kurtosis/commons/testsuite"
 )
 
-type AvaTestSuite struct {}
+type AvaTestSuite struct {
+	ChitSpammerImageName string
+	NormalImageName string
```

Comment:
Per the comment above, this NormalImage should get propagated to every test who wants a normal image